### PR TITLE
Remove the mirrors support from the web site

### PR DESCRIPTION
**Pull request by jsquyres**
Per https://www.mail-archive.com/mirrors@lists.open-mpi.org/msg00174.html, do not merge until July 1, 2018.

### CODE REVIEW

File: papers/papers.inc [hunk 1]
```diff
@@ -8,6 +8,7 @@ if (is_file("data.inc")) {
 
 function paper_page($title, $authors, $presented, $abstract, $files, $mon, $day, $year, $bibtex = "") {
     print("\n\n<h3>Title: $title");
+    date_default_timezone_set('America/Los_Angeler');
```
Comment (bwbarrett) [inline papers/papers.inc]: but shouldn't we at least give it one that's a real timezone?  Also, UTC or death!

Comment (jsquyres) [reply papers/papers.inc]: Hah.  How did PHP not complain about that?

Is the timezone string for UTC `UTC`?

Comment (bwbarrett) [reply papers/papers.inc]: i don't know; that surprised me.  I believe in PHP you can use 'UTC' as a timezone.  Apparently it won't complain either way :).

Comment (jsquyres) [reply papers/papers.inc]: Yes, it looks like it's `UTC`.  I'll update.

**Comment by jsquyres** [comment]
So long, and thanks for all the fish!

**Pull request merged**